packages/shared/src/components/notifications/Toast.tsx

```diff
@@ -0,0 +1,171 @@
+import React, {
+  ReactElement,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { useRouter } from 'next/router';
+import { useQuery, useQueryClient } from 'react-query';
+import {
+  ToastNotification,
+  TOAST_NOTIF_KEY,
+} from '../../hooks/useToastNotification';
+import classed from '../../lib/classed';
+import { Button } from '../buttons/Button';
+import styles from './Toast.module.css';
+import XIcon from '../../../icons/x.svg';
+import { isTouchDevice } from '../../lib/tooltip';
+
+interface ToastProps {
+  autoDismissNotifications?: boolean;
+}
+
+const Container = classed(
+  'div',
+  'fixed left-1/2 flex flex-col justify-center bg-theme-label-primary p-2 rounded-14 border-theme-divider-primary shadow-2',
+  styles.toastContainer,
+);
+const Content = classed('div', 'relative flex flex-row items-center ml-2');
+const Message = classed(
+  'div',
+  'flex-1 mr-2 typo-subhead text-theme-label-invert',
+);
+const Progress = classed(
+  'span',
+  styles.toastProgress,
+  'absolute -bottom-2 h-1 ease-in-out bg-theme-status-cabbage rounded-full',
+);
```

Comment:
Up to you, but perhaps we should move these to a common or util file int he notifications directory? (Just cleaner I think)

non-blocking

Reply:
Great idea. Let me move these ones! Thank you!